test/utils/calculateDimension_spec.js

```diff
@@ -71,4 +71,35 @@ describe('#calculateDimension', () => {
       expect(calculateDimension(testElement, 'width', false, true)).to.equal(108);
     });
   });
+
+  // https://github.com/airbnb/react-dates/issues/1426
+  describe('withMargin false and borderBox true when style properties are absent', () => {
+    let testElement = null;
+
+    beforeEach(() => {
+      const { window } = new JSDOM('<div />');
```

Comment:
i think maybe it'd make more sense to do one mocha run without jsdom, and another with? then `describeIfWindow` would be used to skip the DOM tests for that run.

That would also mean the karma tests wouldn't run with jsdom, since it'd be an entirely different test run.